From 3935ef1d670a4886d82649aaa54263723e5ae72c Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megous@megous.com>
Date: Fri, 10 Nov 2017 13:21:55 +0100
Subject: [PATCH 1039/1089] ARM: dts: sun8i-a83t-tbs-a711: Enable LTR-501-ALS
 IIO

It's already supported in mainline kernel. Though, the driver didn't
have OF compativles table.

Signed-off-by: Ondrej Jirman <megous@megous.com>
---
 arch/arm/boot/dts/sun8i-a83t-tbs-a711.dts | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/arch/arm/boot/dts/sun8i-a83t-tbs-a711.dts b/arch/arm/boot/dts/sun8i-a83t-tbs-a711.dts
index 2427bf3461d7..6acc678d1b5b 100644
--- a/arch/arm/boot/dts/sun8i-a83t-tbs-a711.dts
+++ b/arch/arm/boot/dts/sun8i-a83t-tbs-a711.dts
@@ -250,6 +250,15 @@
 		interrupt-parent = <&pio>;
 		interrupts = <7 10 IRQ_TYPE_EDGE_RISING>; /* PH10 / EINT10 */
 	};
+
+	/* Light Sensor */
+	ltr501: ltr501@23 {
+		status = "disabled"; /* no output */ 
+		compatible = "ltr,ltr501";
+		reg = <0x23>;
+		interrupt-parent = <&pio>;
+		interrupts = <6 12 IRQ_TYPE_EDGE_FALLING>; /* PG12 */
+	};
 };
 
 &mmc0 {
-- 
2.19.2

